Modules/Microsoft365DSC/DSCResources/MSFT_AADAdministrativeUnit/MSFT_AADAdministrativeUnit.psm1

```diff
@@ -100,22 +100,6 @@ function Get-TargetResource
     $nullResult.Ensure = 'Absent'
     try
     {
-        $getValue = $null
-
-        #region resource generator code
-        if (-Not [string]::IsNullOrEmpty($Id))
```

Comment:
Not sure what the reason for this change is here. This is how we should be doing it. If ID is provided, then retrieve it using it, otherwise try the displayname.